core/storage/base_model/gae_models.py

```diff
@@ -259,17 +298,22 @@ def get_multi(cls, entity_ids, include_deleted=False):
             else:
                 none_argument_indices.append(index)
 
-        entities = datastore_services.get_multi(entity_keys)
+        entities = datastore_services.get_multi(entity_keys) # type: List[Optional[SELF_BASE_MODEL]]
         for index in none_argument_indices:
             entities.insert(index, None)
 
         if not include_deleted:
             for i in python_utils.RANGE(len(entities)):
-                if entities[i] and entities[i].deleted:
-                    entities[i] = None
+                entity = entities[i]
+                if entity:
```

Comment:
if entity is not None

Reply:
Done

Reply:
Just confirming, this is for cases like `''` empty string can also escape the if block?